bugbot/rules/webcompat_sightline.py

```diff
@@ -0,0 +1,79 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Any, Optional
+
+from bugbot import gcp
+from bugbot.bzcleaner import BzCleaner
+
+
+class WebcompatSightline(BzCleaner):
+    normal_changes_max = 1000
+    whiteboard_entry = "[webcompat:sightline]"
+
+    def __init__(self):
+        super().__init__()
+        self.autofix_changes = {}
+        self.sightline_ids = []
+
+    def description(self) -> str:
+        return "Web Compat site report in the sightline metric set"
+
+    def filter_no_nag_keyword(self) -> bool:
+        return False
+
+    def has_default_products(self) -> bool:
+        return False
+
+    def get_autofix_change(self) -> dict[str, Any]:
+        return self.autofix_changes
+
+    def handle_bug(
+        self, bug: dict[str, Any], data: dict[str, Any]
+    ) -> Optional[dict[str, Any]]:
+        bug_id = str(bug["id"])
+        whiteboard = bug["whiteboard"]
+
+        if bug["id"] in self.sightline_ids:
+            if self.whiteboard_entry not in whiteboard:
+                self.autofix_changes[bug_id] = {
+                    "whiteboard": whiteboard + self.whiteboard_entry
+                }
+        elif self.whiteboard_entry in whiteboard:
+            self.autofix_changes[bug_id] = {
+                "whiteboard": whiteboard.replace(self.whiteboard_entry, "")
+            }
+
+        return None
```

Comment:
Returning `None` will exclude the bug from the email that will be generated at the end to report the changes to us.

Reply:
That seems fine? This is just syncing a property with an external data source, so sending people an email to say it changed seems unnecessary (unlike the rules where the point is to email people to take action on a set of bugs). I'd hope that for this kind of rule we have normal logging rather than depending on email to individuals as a substitute for logs.

Reply:
Returning `None` for bugs that we do not intend to exclude will break the bot's workflow. See: https://github.com/mozilla/bugbot/blob/a8b33bedcf4c7cf0e0deb5448b2748912ffe9ea7/bugbot/bzcleaner.py#L341-L378

Reply:
OK, I added an email template and made it return the bug when there's a change.